src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -299,7 +306,43 @@ const ChatInfoScreen = () => {
           ListFooterComponent={
             <>
               {isLoadingFetchingChannelDetail ? <ActivityIndicator style={styles.loading} /> : null}
+              {channelInfo?.channelType === CHANNEL_GROUP && (
+                <View style={styles.btnAdd}>
+                  <TouchableOpacity
+                    testID="addParticipant"
+                    onPress={() =>
+                      navigation.push('ContactScreen', {
+                        from: SIGNED,
+                        isAddParticipant: true,
+                        channelId: channelInfo?.id,
+                        existParticipants: channelInfo?.memberUsers?.map((item) => item?.username)
+                      })
+                    }>
+                    <Text style={styles.btnAddText}>+ Add Participants</Text>
+                  </TouchableOpacity>
+                </View>
+              )}
               <View style={styles.gap} />
+              {channelInfo?.channelType === CHANNEL_GROUP && (
+                <View style={styles.actionGroup}>
+                  <TouchableOpacity onPress={onLeaveGroup} style={styles.buttonGroup}>
+                    <View style={styles.imageActContainer}>
+                      <FastImage style={styles.imageAction} source={ExitGroup} />
+                    </View>
+                    <View>
+                      <Text style={styles.textAct}>Exit Group</Text>
+                    </View>
+                  </TouchableOpacity>
+                  <TouchableOpacity onPress={onReportGroup} style={styles.buttonGroup}>
+                    <View style={styles.imageActContainer}>
+                      <FastImage style={styles.imageAction} source={ReportGroup} />
+                    </View>
+                    <View>
+                      <Text style={styles.textAct}>Report Group</Text>
+                    </View>
+                  </TouchableOpacity>
+                </View>
+              )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The code for rendering the "Add Participants", "Exit Group", and "Report Group" buttons is added. These changes are necessary for the new features of adding participants to a group chat, leaving a group chat, and reporting a group chat.
```diff
- <View style={styles.gap} />
+ {channelInfo?.channelType === CHANNEL_GROUP && (
+   <View style={styles.btnAdd}>
+     <TouchableOpacity
+       testID="addParticipant"
+       onPress={() =>
+         navigation.push('ContactScreen', {
+           from: SIGNED,
+           isAddParticipant: true,
+           channelId: channelInfo?.id,
+           existParticipants: channelInfo?.memberUsers?.map((item) => item?.username)
+         })
+       }>
+       <Text style={styles.btnAddText}>+ Add Participants</Text>
+     </TouchableOpacity>
+   </View>
+ )}
+ <View style={styles.gap} />
+ {channelInfo?.channelType === CHANNEL_GROUP && (
+   <View style={styles.actionGroup}>
+     <TouchableOpacity onPress={onLeaveGroup} style={styles.buttonGroup}>
+       <View style={styles.imageActContainer}>
+         <FastImage style={styles.imageAction} source={ExitGroup} />
+       </View>
+       <View>
+         <Text style={styles.textAct}>Exit Group</Text>
+       </View>
+     </TouchableOpacity>
+     <TouchableOpacity onPress={onReportGroup} style={styles.buttonGroup}>
+       <View style={styles.imageActContainer}>
+         <FastImage style={styles.imageAction} source={ReportGroup} />
+       </View>
+       <View>
+         <Text style={styles.textAct}>Report Group</Text>
+       </View>
+     </TouchableOpacity>
+   </View>
+ )}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->